INSTALL.md

```diff
@@ -62,6 +62,9 @@ The GalSim package also requires
   The latter is preferred, since this is now where all future development of 
   this package is happening.
 
+* The future module, which is used to ease compatibility between Python 2
+  and Python 3.
```

Comment:
Elsewhere in INSTALL.md, there are strong statements about requiring python 2.6 or 2.7, so those also have to be updated to say that python 3 is required.